.github/workflows/ci-nix.yml

```diff
@@ -290,9 +290,9 @@ jobs:
             # there's not enough macos runners available for our CI, so test only the more important cross-compilation toolchains
             # if they work, rest probably works as well
           - host: macos
-            toolchain: armv7-linux-androideabi
+            toolchain: armv7-androideabi
```

Comment:
Since `armv7-androideabi` isn't defined in the matrix toolchain, should we either remove this exclude or add it to `matrix.toolchain`?

Reply:
Oh, right.

Reply:
https://github.com/fedimint/fedimint/pull/4337